src/main/java/org/opensearch/commons/authuser/AuthUserRequestBuilder.java

```diff
@@ -42,7 +42,7 @@ public AuthUserRequestBuilder(String auth) {
     }
 
     public Request build() {
-        Request request = new Request("GET", "/_opendistro/_security/authinfo");
+        Request request = new Request("GET", "/_opensearch/_security/authinfo");
```

Comment:
You need to track this down in the security plugin and make sure that this is the change that was made there. I think the path is different.

Reply:
Please open an issue in this repo to change this later, linking to an issue in security to watch.